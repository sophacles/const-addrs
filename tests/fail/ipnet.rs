use const_addrs::ipnet;

fn main() {
    let _ = ipnet!("192.168.1.1/300");
    let _ = ipnet!("2001:db8::32::23/129");
}
