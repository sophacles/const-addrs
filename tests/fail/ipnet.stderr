error: invalid IP address syntax
 --> tests/fail/ipnet.rs:4:20
  |
4 |     let _ = ipnet!("192.168.1.1/300");
  |                    ^^^^^^^^^^^^^^^^^

error: invalid IP address syntax
 --> tests/fail/ipnet.rs:5:20
  |
5 |     let _ = ipnet!("2001:db8::32::23/129");
  |                    ^^^^^^^^^^^^^^^^^^^^^^
